quixstreams/app.py

```diff
@@ -1009,8 +1023,8 @@ def copy(self, **kwargs) -> Self:
         return copy
 
     @property
-    def flush_timeout(self) -> float:
-        return (
+    def flush_timeout(self) -> int:
+        return int(
```

Comment:
I'm unsure if everything should be int or float. Currently most timeout expect an int